src/taipy/core/data/excel.py

```diff
@@ -241,11 +241,17 @@ def _read_as_numpy(self):
         return sheets.to_numpy()
 
     def _do_read_excel(self, engine, sheet_names, kwargs) -> pd.DataFrame:
```

Comment:
What if we change the default value of sheet_names? From this:
```
def _read_as_pandas_dataframe(self, sheet_names=None)
```
To this:
```
def _read_as_pandas_dataframe(self, sheet_names="")
```

And on _do_read_excel, do this:
if sheet_names != "":
            df = pd.read_excel(
                self._path,
                sheet_name=sheet_names,
                **kwargs,
            )
        else:
            df = pd.read_excel(
                self._path,
                **kwargs,
            )

Could that work so we maintain all the options available? If no name is provided, sheet_names is `""` and we just call read_excel without parameters. If the user passes None, them a dict is returned because we would pass `sheet_names=None` to read_excel. And any other name or list of names would alsso work.

The only thing Im questioning is if empty string is a valid excel sheet name 👀 . What do you guys think? @toan-quach  @jrobinAV

Reply:
Actually, I just remembered, when initializing the ExcelDataNode, if sheet_name is not provided in the property, we will set it to None. 
![image](https://github.com/Avaiga/taipy-core/assets/93168955/74a21f03-377f-4018-8e78-d9dc9c14fe5f)

We could set it to the default value of pandas instead ("Sheet1"). This I think is a much simpler solution, reducing a lot of painstaking if-else. The only drawback I can think of atm is we will have to monitor this default value. Though I doubt they have any plans of changing it soon. What do you think? @joaoandre-avaiga @jrobinAV